bifrost_jupyter_extension.json

```diff
@@ -0,0 +1,5 @@
+{
+  "load_extensions": {
+    "bifrost_jupyter_extension/extension": true
+  }
+}
```

Comment:
Sorry if this confuses you. I don't think we need `bifrost_jupyter_extension.json` and `bifrost.json` in our repo.